fury/ui/containers.py

```diff
@@ -220,6 +255,73 @@ def re_align(self, window_size_change):
             msg = "You can only left-align or right-align objects in a panel."
             raise ValueError(msg)
 
+    def update_border_coords(self):
+        """Update the coordinates of the borders
+        """
+        self.border_coords = {'left': (0., 0.),
+                              'right': (1., 0.),
+                              'top': (0., 1.),
+                              'bottom': (0., 0.)}
+
+        for key in self.borders.keys():
+            self.update_element(self.borders[key], self.border_coords[key])
```

Comment:
This method can only be used once during initialisation and I think this can be done in init itself. The coordinates are also hard coded here as well.

This method could be removed.